fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,184 @@
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_id ??
+                          'ID',
+                      content: volunteerInformation.ID,
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_first_name ??
+                          'First Name',
+                      content: volunteerInformation.firstName,
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_last_name ??
+                          'Last Name',
+                      content: volunteerInformation.lastName,
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_email ??
+                          'Email',
+                      content: volunteerInformation.email,
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_phone ??
+                          'Phone',
+                      content: volunteerInformation.mobileNo,
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_prefHours ??
+                          'Preferred Hours',
+                      content: volunteerInformation.prefHours.toString(),
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_expYears ??
+                          'Experience Years',
+                      content: volunteerInformation.expYears.toString(),
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_qualifications ??
+                          'Qualifications',
+                      content: volunteerInformation.qualifications.map((qualification) => qualification.name)
+                                    .join('\n'),
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_possibleRoles ??
+                          'Possible Roles',
+                      content: volunteerInformation.possibleRoles.map((role) => role)
+                                    .join('\n'),
+                    ),
+                    volunteer_information_widget(
+                      title: AppLocalizations.of(context)?.volunteer_availability_monday ??
+                          'Availability Monday',
+                      content: volunteerInformation.availabilities.monday.isEmpty
+                          ? AppLocalizations.of(context)?.unavailable ??
```

Comment:
Please encapsulate the logic to show unavailable text or a formatted hours in a function.